-- Logs begin at Mon 2022-07-11 16:43:39 IST, end at Mon 2022-07-11 16:54:11 IST. --
Jul 11 16:52:50 datacore-PowerEdge-R740 systemd[1]: Started Devstack devstack@c-vol.service.
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]:   from cryptography.utils import int_from_bytes
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: /usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]:   from cryptography.utils import int_from_bytes
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: /opt/stack/cinder/cinder/db/sqlalchemy/models.py:152: SAWarning: implicitly coercing SELECT object to scalar subquery; please use the .scalar_subquery() method to produce a scalar subquery.
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]:   last_heartbeat = column_property(
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: /opt/stack/cinder/cinder/db/sqlalchemy/models.py:160: SAWarning: implicitly coercing SELECT object to scalar subquery; please use the .scalar_subquery() method to produce a scalar subquery.
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]:   num_hosts = column_property(
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: /opt/stack/cinder/cinder/db/sqlalchemy/models.py:169: SAWarning: implicitly coercing SELECT object to scalar subquery; please use the .scalar_subquery() method to produce a scalar subquery.
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]:   num_down_hosts = column_property(
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: CRITICAL cinder [-] Unhandled error: TypeError: <lambda>() takes 0 positional arguments but 1 was given
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder Traceback (most recent call last):
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder   File "/usr/local/bin/cinder-volume", line 10, in <module>
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder     sys.exit(main())
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder   File "/opt/stack/cinder/cinder/cmd/volume.py", line 182, in main
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder     getattr(os_brick, 'setup', lambda: None)(CONF)
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder TypeError: <lambda>() takes 0 positional arguments but 1 was given
Jul 11 16:52:52 datacore-PowerEdge-R740 cinder-volume[3141507]: ERROR cinder 
Jul 11 16:52:53 datacore-PowerEdge-R740 systemd[1]: devstack@c-vol.service: Main process exited, code=exited, status=1/FAILURE
Jul 11 16:52:53 datacore-PowerEdge-R740 systemd[1]: devstack@c-vol.service: Failed with result 'exit-code'.
Jul 11 16:53:08 datacore-PowerEdge-R740 systemd[1]: Started Devstack devstack@c-vol.service.
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]:   from cryptography.utils import int_from_bytes
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: /usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]:   from cryptography.utils import int_from_bytes
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: /opt/stack/cinder/cinder/db/sqlalchemy/models.py:152: SAWarning: implicitly coercing SELECT object to scalar subquery; please use the .scalar_subquery() method to produce a scalar subquery.
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]:   last_heartbeat = column_property(
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: /opt/stack/cinder/cinder/db/sqlalchemy/models.py:160: SAWarning: implicitly coercing SELECT object to scalar subquery; please use the .scalar_subquery() method to produce a scalar subquery.
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]:   num_hosts = column_property(
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: /opt/stack/cinder/cinder/db/sqlalchemy/models.py:169: SAWarning: implicitly coercing SELECT object to scalar subquery; please use the .scalar_subquery() method to produce a scalar subquery.
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]:   num_down_hosts = column_property(
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: CRITICAL cinder [-] Unhandled error: TypeError: <lambda>() takes 0 positional arguments but 1 was given
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder Traceback (most recent call last):
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder   File "/usr/local/bin/cinder-volume", line 10, in <module>
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder     sys.exit(main())
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder   File "/opt/stack/cinder/cinder/cmd/volume.py", line 182, in main
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder     getattr(os_brick, 'setup', lambda: None)(CONF)
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder TypeError: <lambda>() takes 0 positional arguments but 1 was given
Jul 11 16:53:11 datacore-PowerEdge-R740 cinder-volume[3142220]: ERROR cinder 
Jul 11 16:53:11 datacore-PowerEdge-R740 systemd[1]: devstack@c-vol.service: Main process exited, code=exited, status=1/FAILURE
Jul 11 16:53:11 datacore-PowerEdge-R740 systemd[1]: devstack@c-vol.service: Failed with result 'exit-code'.
